src/renderers/wgpu/node_bundle.rs

```diff
@@ -14,10 +14,63 @@ use super::{
     pipeline::{CameraData, InoxPipeline, Uniform},
 };
 
+#[derive(Debug)]
+pub struct PartData(pub RenderBundle, pub Vec<Mask>);
+
+#[derive(Debug)]
 pub enum NodeBundle {
-    Part(RenderBundle, Drawable),
-    Composite(RenderBundle, InoxNodeUuid),
+    Part(PartData),
+    Composite(Vec<PartData>, InoxNodeUuid),
```

Comment:
You could have a `pub struct CompositeData(pub Vec<PartData>, pub InoxNodeUuid);` and then create a `render_composite` like how `render_part` exists.
Although for consistency I think naming them `PartBundle` and `CompositeBundle` is better, `Data` tends to be a tautology very often.